### PR TITLE
Added React router and dev tools

### PR DESCRIPTION
**Pull request by will-nemo**

### CODE REVIEW

File: olwebsite-app/src/components/CarouselWrapper.js [hunk 1]
```diff
@@ -15,18 +15,24 @@ function CarouselWrapper() {
...
+        showStatus={SHOW_STATUS}
+      >
+        <div>
+          <img src={homeLogo} alt="logo" />
+        </div>
+        <div>
+          <img src={homeLogo} alt="logo" />
+        </div>
+        <div>
+          <img src={homeLogo} alt="logo" />
```
Comment (ambid17) [inline olwebsite-app/src/components/CarouselWrapper.js]: are these just for placeholders?

File: olwebsite-app/src/components/Header.js [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
-      <img src={homeLogo} alt="logo" tabIndex={0} />
-      <div tabIndex={0}>Meet the team</div>
-      <div tabIndex={0}>DevDocs</div>
-      <div tabIndex={0}>Blogs</div>
-      <div tabIndex={0}>Resources</div>
-      <div tabIndex={0}>Login</div>
+      <Link to="/">
+        <img className="logo" src={homeLogo} alt="one leif logo" />
+      </Link>
+      <Link className="link" to="/Meet-the-team">
```
Comment (ambid17) [inline olwebsite-app/src/components/Header.js]: maybe a more specific classname? like top-nav-link? If we get more hierarchical in the future this should probably be a standard of sorts

File: olwebsite-app/src/components/CarouselWrapper.js (hunk 1 above)
Comment (zmeriksen) [reply olwebsite-app/src/components/CarouselWrapper.js]: I'd assume so

File: olwebsite-app/src/components/Header.js (hunk 2 above)
Comment (zmeriksen) [reply olwebsite-app/src/components/Header.js]: General Link should be fine... CSS is cascading so we should have a link then also a nav-link for any differences...

File: olwebsite-app/src/components/CarouselWrapper.js (hunk 1 above)
Comment (will-nemo) [reply olwebsite-app/src/components/CarouselWrapper.js]: Yup

**Pull request merged**